### PR TITLE
pls review me 

### PR DESCRIPTION
**Pull request by dustyfingers**

### CODE REVIEW

**Review by ebcitron — COMMENTED** [review]
Everythings looking great, Cant find anything wrong. I appreciate you not seperating your html element names out line  by line. Great job on media break points

File: css/index.css [hunk 1]
```diff
@@ -1,3 +1,299 @@
-/* Use your own code or past solution for Great Idea Web Page CSS here! */
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
 
+html, body, div, span, applet, object, iframe,
```
Comment (ebcitron) [inline css/index.css:6]: I appreciate this

File: css/index.css [hunk 2]
```diff
@@ -1,3 +1,299 @@
...
+
+.copyright {
+	margin: 0 auto;
+	padding: 5rem;
+}
+
+/* media queries */
+
+/* tablet/med size screen */
+@media only screen and (max-width: 800px) {
```
Comment (ebcitron) [inline css/index.css:210]: I personally like to write the media max/min width alongside the rest of the css, that way you can see the other values

**Review by ebcitron — APPROVED** [review]
Awesome

File: css/index.css (hunk 1 above)
Comment (dustyfingers) [reply css/index.css:6]: i know some people have the beautify/prettier extension in vscode and it separates them out line by line, i turned mine off cause it's annoying! haha

**Pull request merged**